test/mocks/databasemock.js

```diff
@@ -143,7 +143,7 @@
 				nconf.set('theme_config', path.join(nconf.get('themes_path'), 'nodebb-theme-persona', 'theme.json'));
 				nconf.set('bcrypt_rounds', 1);
 
-				require('../../build').buildAll(next);
```

Comment:
I think this will cause the build methods to be not tested, we might need to add a separate build.js to the tests folder to test the build code.